collection/rancher/v2.x/profile-collector/continuous_profiling.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
-# Which app to profile? Supported choices: rancher, cattle-cluster-agent
+# Which app to profile? Supported choices: rancher, cattle-cluster-agent, fleet-controller, fleet-agent
 APP=rancher
 
+# Namespace  of the app, optional
```

Comment:
```suggestion
# Namespace  of the app, auto-detected from the app name by default
```

Reply:
removed the declaration of namespace, in favor of auto-detecting the namespace on the collect function